lib/SyTest/Federation/Room.pm

```diff
@@ -385,14 +385,21 @@ sub make_join_protoevent
       $self->get_current_state_event( "m.room.create" ),
       $self->get_current_state_event( "m.room.join_rules" ),
    );
+   my $auth_events = $self->make_event_refs( @auth_events );
+
```

Comment:
Do we want to do the same checks for `auth_events`?

Reply:
I guess.

Reply:
actually no, that would be redundant. there's already a `grep defined` at line 384.